src/main/java/uk/gov/justice/digital/hmpps/keyworker/services/KeyworkerService.java

```diff
@@ -146,7 +146,15 @@ private static String getKeyWorkerFullName(final KeyworkerDto keyworkerDto) {
                         .staffId(staffDetail.getStaffId())
                         .email(staffDetail.getEmail())
                         .build());
+
             }
+            return Optional.empty();
+        }
+        final var detail = nomisService.getPrisonerDetail(offenderNo, false);
```

Comment:
don't think should need `admin` here to get prisoner details.  If we do then we won't go to nomis anyway...